src/components/admin/Tables/UsersTable.js

```diff
@@ -100,6 +101,25 @@ export default function UsersTable(): React.Node {
     setSelectedDelete(-1);
   };
 
+  const handleConfirmDelete = async () => {
+    try {
+      await deleteDocument('active-members', 'count', selectedDelete);
+      setSelectedDelete(-1);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const saveNote = async (user: UserType, note: string) => {
+    try {
+      await updateAdminNote('email', user.email, note);
+    } catch (error) {
+      console.log(error);
```

Comment:
If this is final version of this function, maybe some pop-up will be better than console log from user experience point of view?

Reply:
yes, I have that in mind, but I want to add it later on as a feature.